tests/accesscontrol/tests/access_control_pod_role_bindings.go

```diff
@@ -0,0 +1,131 @@
+		Expect(err).ToNot(HaveOccurred())
+
+		err = namespaces.Clean(tsparams.TestAnotherNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+		setupInitialRbacConfiguration()
+	})
+
+	It("one pod with valid role binding", func() {
+		By("Define pod")
+
+		testPod := pod.DefinePod(tsparams.TestPodName, tsparams.TestAccessControlNameSpace,
+			globalhelper.Configuration.General.TestImage, tsparams.TestDeploymentLabels)
+
+		pod.RedefineWithServiceAccount(testPod, tsparams.TestServiceAccount)
+
+		err := globalhelper.CreateAndWaitUntilPodIsReady(testPod, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start pod-role-bindings")
+		err = globalhelper.LaunchTests(
+			tsparams.TnfPodRoleBindings,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TnfPodRoleBindings,
+			globalparameters.TestCaseSkipped)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("one pod with service account in different namespace", func() {
+		By("Define pod")
+
+		testPod := pod.DefinePod(tsparams.TestPodName, tsparams.TestAccessControlNameSpace,
+			globalhelper.Configuration.General.TestImage, tsparams.TestDeploymentLabels)
+
+		pod.RedefineWithServiceAccount(testPod, tsparams.TestServiceAccount)
+		err := globalhelper.CreateAndWaitUntilPodIsReady(testPod, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create namespace")
+		err = namespaces.Create(tsparams.TestAnotherNamespace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred(), "Error creating namespace")
+
+		// Delete service account
+		err = globalhelper.DeleteServiceAccount(tsparams.TestServiceAccount, tsparams.TestAccessControlNameSpace)
+		Expect(err).ToNot(HaveOccurred())
+		// Create the service account in a new namespace
+		err = globalhelper.CreateServiceAccount(tsparams.TestServiceAccount, tsparams.TestAnotherNamespace)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start pod-role-bindings")
+		err = globalhelper.LaunchTests(
+			tsparams.TnfPodRoleBindings,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).NotTo(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TnfPodRoleBindings,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("one pod with role binding in different namespace", func() {
+		By("Define pod")
+		testPod := pod.DefinePod(tsparams.TestPodName, tsparams.TestAccessControlNameSpace,
+			globalhelper.Configuration.General.TestImage, tsparams.TestDeploymentLabels)
+
+		pod.RedefineWithServiceAccount(testPod, tsparams.TestServiceAccount)
+		err := globalhelper.CreateAndWaitUntilPodIsReady(testPod, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Create namespace")
```

Comment:
Might make sense to create this namespace in a BeforeAll in this file, as in https://github.com/test-network-function/cnfcert-tests-verification/blob/411e768b52982dbada8341d4423455f070037db2/tests/accesscontrol/tests/access_control_namespace.go#L17

Reply:
This ns is required for only this set of test cases. But access-control has so many test cases. We do not need to create this NS for them.

Reply:
Oh sorry I meant that defining the namespace in a BeforeAll could happen inside this file, just before the BeforeEach for example, so that you don't have to repeat the creation in both cases that use it. Either way works, might just be a bit cleaner to only create it once